subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/m8/ToolingApiLoggingCrossVersionSpec.groovy

```diff
@@ -148,6 +148,7 @@ project.logger.debug("debug logging");
     private ExecutionResult runUsingCommandLine() {
         def executer = targetDist.executer(temporaryFolder, getBuildContext())
             .requireGradleDistribution()
+            .withPartialVfsInvalidation(false) // Don't show incubating message for logging tests
```

Comment:
Again, would be nice to ensure that this gets removed later. Maybe add a comment to `VirtualFileSystemPartialInvalidationIntegrationTest.incubating message is shown for partial invalidation`?

Reply:
Like above, I think the `withPartialVfsInvalidation` should be removed completely. If we do so, then this usage will also go away.